src/libs/ComponentSpecs.js

```diff
@@ -262,10 +262,16 @@ const prototypeSpecs = {
       options.margins = margins;
     }
     const reactI13n = this._getReactI13n();
+    const containerId = this._i13nNode._model.scrollableContainerId;
```

Comment:
is it possible to use `this.props.i13nScrollableContainerId`, similar to how `viewport` is passed in as `this.props.viewport`?